sp800-63a/sec5_proofing.md

```diff
@@ -193,27 +193,23 @@ The following requirements apply to the identity verification steps for IAL 2 an
 
 
 
-- The CSP SHOULD verify knowledge of recent transactional history that the CSP is a participant to.  The CSP SHALL ensure transaction information meets the minimum entropy for a Memorized Secret. For example, verification of amount and confirmation number of a micro-deposit to a claimed and valid bank account.
-
-
-
-- The CSP MAY perform KBV by asking questions of the claimed identity to demonstrate they are the owner of the claimed information. 
+- The CSP SHOULD perform KBV by verifying knowledge of recent transactional history that the CSP is a participant to.  The CSP SHALL ensure transaction information meets the minimum entropy for a Memorized Secret. For example, verification of amount and confirmation number of a micro-deposit to a claimed and valid bank account. The CSP MAY perform KBV by asking questions of the claimed identity to demonstrate they are the owner of the claimed information. However, the following requirements apply:
```

Comment:
You should be able to just fetch the changes from your GitHub repository into your usual laptop; it's all there.